### ISSUE TITLE
Mistake in "Deep Learning with PyTorch: A 60 Minute Blitz"

### ISSUE THREAD

**Issue by JianLiu91** [pytorch/pytorch#1871]
In neural networks tutorial:

http://pytorch.org/tutorials/beginner/blitz/neural_networks_tutorial.html

Just below the picture,  obviously "weight = weight + learning_rate * gradient" should be "weight = weight - learning_rate * gradient", which is the gradient *descent* algorithm.

**Comment by ilija139** [comment]
Actually, it doesn't matter. The _descent_ in the name is with respect to the loss, i.e. it's minimizing the loss and not maximizing.

**Comment by soumith** [comment]
fixed via https://github.com/pytorch/tutorials/commit/01b89c9387203dddbfbaeb9dce047df4db62d22c
